lib/phoenix/channel/server.ex

```diff
@@ -147,9 +147,9 @@ defmodule Phoenix.Channel.Server do
   def reply(pid, ref, topic, {status, payload}, serializer)
       when is_binary(topic) and is_map(payload) do
 
-    send pid, serializer.encode!(
+    send pid, {:channel_push, serializer.encode!(
```

Comment:
Do you remember why we had to wrap it here? I know we discussed it by I don't remember why. :)